applications/nrf5340_audio/src/modules/dfu_entry.h

```diff
@@ -8,9 +8,15 @@
 #define _DFU_ENTRY_H_
 
 /**
- * @brief Check Btn pressed status to advertise SMP_SVR service only
+ * @brief	Function pointer of BLE subsystem initialization API
  */
+typedef int (*ble_init_func_t)(void *ready_callback);
```

Comment:
```suggestion
typedef int (*ble_init_func)(void *ready_callback);
```

We dont use '_t' in these cases

Reply:
done

Reply:
done